src/test/java/com/wikia/webdriver/elements/oasis/pages/TemplatePage.java

```diff
@@ -69,4 +70,10 @@ public String getPageBackgroundColor() {
 
     return pageBackgroundColor.getCssValue("background-color");
   }
+
+  public TemplatePage createTemplate(String templateName) {
+    new ArticleContent().push("", "Template:" + templateName);
```

Comment:
I would advice you to add a separate `TemplateContent` class, which will push wikitext always to `Template` namespace